test/passing/comments_in_record.ml

```diff
@@ -21,6 +21,6 @@ let { (* cmts *)
     ; d=
       (* d *)
       (D: loooooooooooooooooooooooooooooooooooooooooooooooooooooooong_int)
-        (* d *)
-    ; e : loooooooooooooooooooooooooooooooooooooooooooooooooooooooong_int } =
+    ; (* e *)
+      e : loooooooooooooooooooooooooooooooooooooooooooooooooooooooong_int } =
```

Comment:
@hhugo, I had to adjust this, and the .ref file below, while rebasing. Could you check that this still performs the intended test?

Reply:
it's ok.